codegen/projections/white_label/lib/white_label/errors.rb

```diff
@@ -84,5 +84,29 @@ def initialize(http_resp:, **kwargs)
       attr_reader :data
     end
 
+    module EventStream
```

Comment:
Does it make sense to nest these? Shouldn't we make the service errors transport agnostic (and potentially event/response agnostic) instead?